# Goal
bool double_calibrate
---
# Result
bool success
string message
---
# Feedback
float32 progress # 0.0-1.0 indicating calibration progress
